packages/forma-36-react-components/src/alpha.ts

```diff
@@ -1,3 +1,4 @@
 export { Autocomplete } from './components/Autocomplete/Autocomplete';
 export { Grid } from './components/Grid';
 export { GridItem } from './components/Grid/GridItem';
+export { NavigationIcon } from './components/NavigationIcon/NavigationIcon';
```

Comment:
I am not sure if this should be alpha 🤔

Reply:
Me neither... I would say "yes", maybe ask the other Forma team people (????)

Reply:
other Forma team people! Please help! 😄
@m10l @mshaaban0 WDYT?

Reply:
I'd say yes to it being alpha :)